gbfs.md

```diff
@@ -267,7 +331,7 @@ Field Name | Required | Type | Defines
 
 ### vehicle_types.json *(added in v2.1-RC)*
 
-The following fields are all attributes within the main "data" object for this feed.
+This file should be published by systems offering multiple vehicle types for rental, for example pedal bikes and ebikes. <br/>The following fields are all attributes within the main "data" object for this feed.
```

Comment:
I think it'd be good to add stronger wording here to reflect the requirement of this file in these types of systems. Maybe something like:

> This file **must** be published...

Reply:
@evansiroky I'm reluctant to make `vehicle_types.json` a requirement when this is intended to be a non-breaking PR. While requiring it may not break down stream apps it seems like a significant change for producers. Currently it's conditionally required of systems who publish vehicle type data in `free_bike_status.json`. I think we should revisit how it's defined in v3.0.

Reply:
I'm confused. In the [Files](https://github.com/NABSA/gbfs/blob/best-practices-nb/gbfs.md#files) section, it says that this file is conditionally required already. Seems like that same language ought to be used here to be consistent.

<img width="1206" alt="Screen Shot 2020-11-02 at 10 10 00 AM" src="https://user-images.githubusercontent.com/3112493/97903279-a113eb80-1cf3-11eb-9d24-9f69f6192275.png">

Reply:
I've added the language from the Files section-

As it's currently defined under Files,  systems publishing vehicle type information in `free_bike_status` MUST publish `vehicle_types.json`. If they are not including vehicle type information in `free_bike_status` they would be exempted from that requirement regardless of the types of vehicles offered.  This says that for those that operate multiple types of vehicles, publishing vehicle_types.json is something they SHOULD do.  If that's not adequate, we could redefine `vehicle_types.json` in v3.0 to require it of systems operating multiple types of vehicles